inst/en_gb.dic

```diff
@@ -0,0 +1,19 @@
+Github
+ICT
+IFBL
+INBOVEG
+Inc
+MILKLIM
+Natura
+PPA
+Rstudio
+Sublicensing
+WIPO
+defenses
+favor
```

Comment:
Yes , but "favor" is in the LICENSE file if I remember well. So I did not want to change that as it will trigger an alarm from checklist. If it needs to be changed, it would need to be changed in the template license files included in the checklist package.